ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__automated-release-stable-4.12-upgrade-from-stable-4.12.yaml

```diff
@@ -61,19 +61,19 @@ tests:
   cron: 8 8 29 2 *
   steps:
     allow_skip_on_success: true
-    cluster_profile: aws-autorelease-qe
+    cluster_profile: aws-sustaining-autorelease-412
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
+      BASE_DOMAIN: sustaining-aws-412.devcluster.openshift.com
     test:
     - chain: openshift-upgrade-qe-sanity
     workflow: cucushift-installer-rehearse-aws-ipi-minimal-permission
 - as: azure-ipi-mini-perm-fips-f999
   cron: 8 8 29 2 *
   steps:
     allow_skip_on_success: true
-    cluster_profile: azure-autorelease-qe
+    cluster_profile: azure-sustaining-autorelease-412
     env:
-      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      BASE_DOMAIN: sustaining_az_412.azure.devcluster.openshift.com
```

Comment:
`baseDomain: Invalid value: "sustaining_az_412.azure.devcluster.openshift.com": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')`